frontend/src/elements/Main.jsx

```diff
@@ -36,6 +36,7 @@ export function Main() {
             } else {
                 let bigX = -1
                 let bigY = -1
+
```

Comment:
What? xd

Reply:
branched before removing the console.log so i missed removing the line in this branch